### PR TITLE
[T5A1][W11-B4] Charlton Lim

### PR DESCRIPTION
**Pull request by cadmusthefounder**

### CODE REVIEW

**Comment by cadmusthefounder** [comment]
Ready for review.

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -74,12 +75,32 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Edit tag
+
+**MSS**
+
+1. User requests to edit tag "y" to tag "x"
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: - what happens if "x" is same as"y"?
- what happens if "x" is already present in the system?

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -74,12 +75,32 @@ Use case ends.
...
+> 1a1. AddressBook shows an error message
+  Use case resumes at step 1
+
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be scalable.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: how scalable?
- In general it is better to be as specific and objective as possible.
- words like less, more, fast, etc., are subjective

**Comment by okkhoy** [comment]
@cadmusthefounder 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**